packages/react-native/React/Fabric/Mounting/ComponentViews/ScrollView/RCTPullToRefreshViewComponentView.mm

```diff
@@ -148,7 +148,9 @@ - (void)didMoveToWindow
 {
   [super didMoveToWindow];
   if (self.window) {
-    [self _attach];
+    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Can you use [RCTExecuteOnMainQueue](https://github.com/facebook/react-native/blob/main/packages/react-native/React/Base/RCTUtils.h#L41) instead? It avoids the jump if we are already on the main queue.

Reply:
It's all about placing the block at the end of the queue - see my response here: https://github.com/facebook/react-native/pull/48580#issuecomment-2582382187